cmd/statusd/main.go

```diff
@@ -24,8 +24,11 @@ import (
 	"github.com/status-im/status-go/profiling"
 )
 
+const (
+	serverClientName = "Statusd"
```

Comment:
lowercase maybe, to be in line with comments and PR description? `statusd`

Reply:
haha, just saw the commit `make it capitalized`

Reply:
All client names that I found are capitalized so I went that way. Not sure about it though.